Functions.Templates/Templates/SqlInputBinding-JavaScript/readme.md

```diff
@@ -0,0 +1,9 @@
+# SQL Input Binding - JavaScript
+
+The `SQL Input Binding` makes it easy to retrieve data from a database, returning the output of the query or stored procedure to the function.
+
+## How it works
+
+For a `SQL Input Binding` to work, you can provide the query to retrive data from an existing object in the database. For instance, you can set the query to `Select * From [dbo].[table1]` to query `[dbo].[table1]` object in the commandText attribute.
```

Comment:
I copied the readme from here to be consistent
https://github.com/Azure/azure-functions-templates/blob/d606cfcb697c3eff1a63eb52e65d1377120d6db9/Functions.Templates/Templates/SqlInputBinding-CSharp/readme.md?plain=1#L7

Reply:
@MaddyDev What @lucyzhang929 suggested would be good. What you have right now mentions the commandText attribute which isn't accurate for non-C# languages (as they don't use the attribute)

Reply:
And I'd actually suggest using what I just pushed up for TS to be clearer 

```
For a `SQL Input Binding` to work, you can provide the query to retrieve data from an existing object in the database. For instance, you can set the query to `SELECT * FROM [dbo].[table1]` in the `commandText` value in your `function.json` to return all the values from the `[dbo].[table1]` table in the database.
```

Reply:
Updated it.